Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertZone.cs

```diff
@@ -0,0 +1,69 @@
+using Autodesk.Revit.DB;
+using Objects.BuiltElements;
+using Speckle.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Objects.BuiltElements.Revit;
+using DB = Autodesk.Revit.DB.Mechanical;
+using Point = Objects.Geometry.Point;
+
+namespace Objects.Converter.Revit
+{
+  public partial class ConverterRevit
+  {
+    public ApplicationObject ZoneToNative(Zone speckleZone)
+    {
+      var revitZone = GetExistingElementByApplicationId(speckleZone.applicationId) as DB.Zone;
+      var appObj = new ApplicationObject(speckleZone.id, speckleZone.speckle_type)
+      {
+        applicationId = speckleZone.applicationId
+      };
+
+      return appObj;
+    }
+
+    public BuiltElements.Revit.Zone ZoneToSpeckle(DB.Zone revitZone)
+    {
+      var speckleZone = new Zone
+      {
+        name = revitZone.Name,
+        area = GetParamValue<double>(revitZone, BuiltInParameter.ROOM_AREA),
+        volume = GetParamValue<double>(revitZone, BuiltInParameter.ROOM_VOLUME),
+        perimeter = GetParamValue<double>(revitZone, BuiltInParameter.ZONE_PERIMETER),
+        serviceType = revitZone.ServiceType.ToString()
+      };
+
+      // Optionally Zones and Spaces could be sent as a Parent Child, but edge cases abound.
+      // var spaces = new List<DB.Space>();
+      // var zoneSpaces = revitZone.Spaces.GetEnumerator();
+      //
+      // while (zoneSpaces.MoveNext())
+      // {
+      //   if (zoneSpaces.Current is DB.Space space)
+      //   {
+      //     spaces.Add(space);
+      //   }
+      // }
+      // speckleZone.spaces = spaces.Select(x => SpaceToSpeckle(x)).ToList();
```

Comment:
curious what the edge cases are. This method (parent references all children) seems more consistant with how the rest of our objects are structured, as opposed to the current implementation (each child references parent).

That being said, I have no idea what zones and spaces are even used for, so there may be a very good reason to invert that relationship.

Reply:
Zones only exist in the context of Spaces existing. Spaces don't require a Zone to exist to be used.

What this child > parent relationship pattern enables us to not:
1. Have some Spaces in a category of their own but some as children of Zones (or worse, duplicated references)
2. Break existing workflows expecting Spaces in a category

It also allows for:
- Retain the existing method in SpaceToNative of allocating to an existing Zone. 
- This can guarantee the Zone exists before adding to its collection and not need to worry about what order elements exist in a commit.

I'm relaxed about doing it differently, it's clear the conversion part is easy.